packages/in_app_purchase/lib/src/in_app_purchase_connection/google_play_connection.dart

```diff
@@ -39,6 +42,8 @@ class GooglePlayConnection
     }
   }
 
+  @override
+  Future<List<Product>> getProductList(List<String> identifiers) => null;
```

Comment:
nit: Would prefer `throw UnimplementedError` to make it really clear that this isn't done yet.

https://api.dartlang.org/stable/2.1.0/dart-core/UnimplementedError-class.html